auto_nag/scripts/bisection_without_regressed_by.py

```diff
@@ -121,34 +134,83 @@ def get_bz_params(self, date):
             "o2": "everchanged",
             "n3": 1,
             "f3": "longdesc",
-            "o3": "casesubstring",
-            "v3": "his bug contains a bisection range, ",
+            "o3": "substring",
+            "v3": " this bug contains a bisection range",
             # TODO: Nag in the duplicate target instead, making sure it doesn't already have regressed_by.
             "f4": "resolution",
             "o4": "notequals",
             "v4": "DUPLICATE",
-            "emaillongdesc1": "1",
-            "emailtype1": "exact",
-            "email1": "bugmon@mozilla.com",
+            "f5": "delta_ts",
+            "o5": "greaterthan",
+            "v5": self.oldest_comment_date,
+            "f6": "OP",
+            "j6": "OR",
+            "f7": "commenter",
+            "o7": "equals",
+            "v7": "bugmon@mozilla.com",
+            "f8": "longdesc",
+            "o8": "substring",
+            "v8": "mozregression",
+            "f9": "CP",
         }
 
+    @staticmethod
+    def is_mozregression_analysis(comment_text: str):
+        """Check if the comment has a regression range from mozregression."""
+        return "ozregression" in comment_text and "pushloghtml" in comment_text
```

Comment:
mozregression is also used for finding fixes, e.g. see https://bugzilla.mozilla.org/show_bug.cgi?id=1334853#c1.
We could ignore the comment if it contains the string "find-fix" or "First good revision" or "Last bad revision" (if it was used to find a regression, it would have been "Last good revision" and "First bad revision").

Reply:
Done in 448c0dd87a45025918821377b45bf82452b9dd82